mantidproject/mantid
# Add different groups to Reflectometry (Polref) GUI

Adds different groups of runs to the Reflectometry GUI.

**To test:**

- Code changes. Note that the first 11 commits come from branch `17817_Label_groups_with_run_title`
- Functional testing: 
  - Open Mantid Plot -> Interfaces -> Reflectometry -> ISIS Reflectometry (Polref).
  - In the "Runs" and "Settings" tabs, check that there are two different groups with the same widgets.
  - Go to "Runs" tab. Check that actions accessible from the menus (i.e. "Reflectometry" and "Edit") apply to the selected table. For instance, loading a .tbl file should populate the table in the current group. The same should happen with the other options (new table, insert row, delete row, etc).
  - Check that both tables are able to reduce data.
  - Enter some values in "Settings". Check that the table in the first groups takes the settings from the first group in "Settings". The same should happen for the second group.

Fixes #18119 

Release notes were updated: https://github.com/mantidproject/mantid/pull/18124/commits/1409ebf8e8d7f9595cd53f5e48fa28c131f0e68f

Documentation will be updated in a separate issue before the next release: https://github.com/mantidproject/mantid/issues/18176

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
